packages/design-system/src/components/Dialog/Dialog.tsx

```diff
@@ -334,86 +148,90 @@ export const Dialog = (props: DialogProps) => {
     };
   }, []);
 
-  const {
-    actions,
-    actionsClassName,
-    analytics,
-    ariaCloseLabel,
-    children,
-    className,
-    closeButtonSize,
-    closeButtonText,
-    closeButtonVariation,
-    closeIcon,
-    closeText,
-    escapeExits,
-    escapeExitDisabled,
-    headerClassName,
-    heading,
-    onExit,
-    size,
-    title,
-    ...modalProps
-  } = props;
+  // Toggle dialog open
+  const dialogRef = useRef(null);
+  const lastActiveElement = React.useRef(null);
+  const firstRender = React.useRef(true);
+
+  useLayoutEffect(() => {
+    if (window.HTMLDialogElement === undefined) {
+      dialogPolyfill.registerDialog(dialogRef.current);
+      // import('dialog-polyfill/dist/dialog-polyfill.css');
```

Comment:
This is where I'd like to conditionally import the polyfill CSS. The error seemed to be this file couldn't find this CSS path and I didn't know how to fix that.

Reply:
This is only 37 lines of CSS unminified, so I'm not worried about it. Do you think when we implement it for real that it would work to import it from our SCSS files? That is, I wonder if we can avoid changing the webpack config if we forget about conditionally importing it.

Reply:
That is, unless the concern is that we shouldn't be applying that CSS at all if it has native support

Reply:
My concern is if we import the polyfill CSS regardless of IE/not IE and the styles get updated (unlikely but not impossible), we might see visual regressions get into our component. We could skirt that issue altogether and copy those styles into our `dialog.scss` file (at the top) and not bother importing at all.